intel_extension_for_transformers/llm/runtime/graph/scripts/run_llm.py

```diff
@@ -17,6 +17,7 @@
 import argparse
 from typing import List, Optional
 import subprocess
+from transformers import AutoTokenizer
```

Comment:
better prepare sth like ```requirements.txt``` since you add new dependency

Reply:
Done. graph/requirements.txt has included the transformer already.